app/assets/stylesheets/common/base/discourse.scss

```
@@ -853,3 +989,124 @@ a#skip-link {
   overflow: hidden !important;
   margin-right: var(--scroll-gap, 0);
 }
+
+body.has-sidebar-page {
+  .wrap {
+    // increase page max-width to accommodate sidebar width
+    max-width: calc(var(--d-sidebar-width) + var(--d-max-width));
+  }
+
+  #main-outlet-wrapper {
+    grid-template-columns: var(--d-sidebar-width) minmax(0, 1fr);
+    gap: 0 2em;
+    padding-left: 0;
+
+    @include viewport.until(lg) {
+      gap: 0 1em;
+    }
+  }
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  body.sidebar-animate {
+    #main-outlet-wrapper {
+      transition-property: grid-template-columns, max-width;
+      transition-timing-function: var(--d-sidebar-animation-ease);
+      transition-duration: var(--d-sidebar-animation-time);
+    }
+
+    .d-header-wrap .wrap {
+      transition: max-width var(--d-sidebar-animation-time)
+        var(--d-sidebar-animation-ease);
+    }
+  }
+}
+
+.mobile-view {
+  .boxed {
+    .contents {
+      // adds space below avatar
+      padding: 10px 0 0 0;
+    }
+  }
+}
+
+.ios-device {
+  textarea {
+    background-color: var(--secondary);
+    -webkit-tap-highlight-color: transparent;
+  }
+
+  input#reply-title {
+    -webkit-tap-highlight-color: transparent;
+  }
+}
+
+// todo: replace — legacy, currently only seems to be used for assigns
+.mobile-nav {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  position: relative;
+
+  a {
+    color: var(--primary);
+
+    .d-icon {
+      margin-right: 0.25em;
+      color: var(--primary-medium);
+    }
+  }
+
+  > li > a.expander {
+    display: flex;
+    align-items: center;
+
+    @include form-item-sizing;
+    border-color: var(--primary-medium);
+    max-width: 100%;
+
+    .selection {
+      @include ellipsis;
+      max-width: 120px;
+    }
```

a bit of mixin positioning consistency

```suggestion
  > li > a.expander {
    display: flex;
    align-items: center;
    border-color: var(--primary-medium);
    max-width: 100%;

    @include form-item-sizing;

    .selection {
      max-width: 120px;

      @include ellipsis;
    }
```